testing.sagebrush.master/Stepwat_Inputs/Input/sxw/sxwprod_v2.in

```diff
@@ -1,30 +1,28 @@
 # sxw prod definition file STEPPEWAT
-
-#Production Coefficients for STEPWAT
+# Production Coefficients for STEPWAT
 
 # These values are used to convert yearly biomass
 # from STEPPE to monthly production values for SOILWAT
 
+######################### LITTER ###########################
 # LITTER - use the maximum value from the converted biomass 
 #          to multiply by this number to compute a litter 
 #          value in g/m^2.  Needn't sum to 1.0.
```

Comment:
Maybe update comment to explain that litter fraction values are multiplied with monthly biomass (instead of [annual] maximum values as previously)?

Reply:
@dschlaep I think you mean update to indicate these monthly fractions are multipled by annual biomass instead of monthly biomass (previous implementation)?

Reply:
@kpalmqui Yes, you are correct! I got it backwards, sorry!

Reply:
thank you! addressed by 23d9f23